src/drivers/imx/ipc.c

```diff
@@ -171,12 +171,6 @@ struct ipc_data_host_buffer *ipc_platform_get_host_buffer(struct ipc *ipc)
 
 int platform_ipc_init(struct ipc *ipc)
 {
-	struct task_ops ops = {
-		.run = ipc_platform_do_cmd,
-		.complete = ipc_platform_complete_cmd,
-		.get_deadline = ipc_task_deadline,
-		.is_ready = NULL,
-	};
```

Comment:
@tlauda this is getting too confusing. You add this struct in one of the patches in the series only to be removed later on?
I think a better way of doing this is to use the common ipc task ops struct from the beginning

Reply:
Done.